ttnn/cpp/ttnn/operations/ccl/reduce_scatter/reduce_scatter_pybind.cpp

```diff
@@ -63,6 +66,7 @@ void py_bind_reduce_scatter(pybind11::module& module) {
         Keyword Args:
             * :attr:`num_links` (int): Number of links to use for the all-gather operation.
             * :attr:`memory_config` (Optional[ttnn.MemoryConfig]): Memory configuration for the operation.
+            * :attr:`topology` (Optional[ttnn.ccl.topology]): The topology configuration to run the operation in. Valid options are Ring and Linear. Default = Ring
```

Comment:
Please align with @yan-zaretskiy om the new docs format. This op seem to be using the old format.

Reply:
Correct - the CCL ops have not been updated. I've opened an issue here to track: https://github.com/tenstorrent/tt-metal/issues/13249